CHANGELOG.md

```diff
@@ -2,6 +2,92 @@
 
 All notable changes to this project will be documented in this file.
 
+## Unreleased - as of Sprint 44 end 2016-08-1
+
+### [Added](https://github.com/ManageIQ/manageiq/issues?q=milestone%3A%22Sprint+44+Ending+Aug+1%2C+2016%22+label%3Aenhancement)
+
+- Automate
+  - Simulation: RBAC filtering applied to Object Attributes
+  - Service Provisioning: Exposed number_of_vms when building the provision request for a service
+  - Service Dialogs: Support for “visible” flag for dynamic fields
+  - Expose Compliance and ComplianceDetail models
+  - New associations on VmOrTemplate and Host models:
+    - `expose :compliances`
+    - `expose :last_compliance`
+  - New Service Models
+    - Compliance: `expose :compliance_details`
+    - ComplianceDetail: `expose :compliance`, `expose :miq_policy`
+  - Generic Object: Service models created for GenericObject and GenericObjectDefinition
+- Platform
+  - PostgreSQL High Availability
+    - Added [repmgr](http://repmgr.org/)  to support automatic failover
+    - Maintain list of active standby database servers
+    - Added [pg-dsn_parser](https://github.com/ManageIQ/pg-dsn_parser) for converting DSN to a hash
+  - Tenancy: Added parent_id to CloudTenant as prerequisite for mapping OpenStack tenants to ManageIQ tenants
+  - Watermark reports updated to be based on max of daily max value instead of max of average value
+  - Nice values added to worker processes
+-  Providers
+  - Google Compute Engine: Metrics
+  - Hawkular
+    - Operations: Add New and Start/stop deployment
+    - Performance reports for datasources
+    - Collect more metrics for datasource
+  - Kubernetes: Cross-linking with OpenStack instances
+  - Microsoft Azure:Support floating IPs during provisioning
+  - Nuage: Inventory of Managed Cloud Subnets
+  - Red Hat Enterprise Virtualization: v4 API
+  - VMware vSphere: Storage Profiles modeling and inventory
+  - VMware vCloud: Initial PRs for modeling and Inventory
+  - Pluggability: SupportsFeatureMixin
```

Comment:
I think it's good to mention things that we discussed at the Summit ... but, maybe that should be separated out.

However, for the purpose of this changelog, I'm fine with pulling this out of the changelog.